viral/offline_sequence_detection.py

```
@@ -0,0 +1,940 @@
+                grosmark_config,
+                bayesian_config,
+            )
+            ppm.append(posterior_probability_matrix)
+        #     if check_significance_pse_event(posterior_probability_matrix):
+        #         significant_events.append(posterior_probability_matrix)
+        # print(f"Found {len(significant_events)} significant events")
+        # TODO: this certainly is wrong so change later
+        calculate_circular_weighted_correlations(
+            events_ppm=ppm,
+        )
+        1 / 0
```

Ruff (0.14.5)

Found useless expression. Either assign it to a variable or remove it.

(B018)